src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -135,6 +144,92 @@ def apply_quantization_config(model: Module, config: QuantizationConfig):
     apply_quantization_status(model, config.quantization_status)
 
 
+def _scheme_from_targets(
```

Comment:
let's rename to merge schemes to make this more clear

Reply:
any reason not to set target to the layer name? on save we generate new schemes anyways with deduplication